common/src/main/java/com/wynntils/utils/render/Texture.java

```diff
@@ -199,7 +199,11 @@ public enum Texture {
     HIGHLIGHT("ui_components/highlight.png", 256, 256),
     OVERLAY_SELECTION_GUI("ui_components/overlay_selection_gui.png", 195, 256),
     SCROLL_BACKGROUND("ui_components/scroll_background.png", 294, 198),
-    VIGNETTE("ui_components/vignette.png", 512, 512);
+    VIGNETTE("ui_components/vignette.png", 512, 512),
+    // endregion
+
+    // region World
+    LEADERBOARD_BADGES("world/leaderboard_badges.png", 256, 256);
```

Comment:
I dunno how I feel about this being under World. Legacy had this + lootrun paths under world, but lootrun paths are moved in Artemis. Imo this might be a better fit under icons or a new category called "players"